sentry-android-core/lint-baseline.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 7.2.0" type="baseline" client="gradle" dependencies="true" name="AGP (7.2.0)" variant="all" version="7.2.0">
+
+    <issue
+        id="TrulyRandom"
```

Comment:
Needed because we can't use `SuppressLint` on Java projects, and it generates 1 per project.